tracing-syslog/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+#[cfg(unix)]
+mod syslog;
+#[cfg(unix)]
+pub use syslog::*;
```

Comment:
It'd be great to have doc comments with examples and tests here.

Reply:
I've never been sure about this: for a crate that has a primary type (like `Syslog` in this case), should the majority of the docs go with that type or in `lib.rs`?